packages/playwright-core/src/server/page.ts

```diff
@@ -1027,7 +1027,7 @@ async function snapshotFrameForAI(progress: Progress, frame: frames.Frame, frame
   const lines = snapshot.split('\n');
   const result = [];
   for (const line of lines) {
-    const match = line.match(/^(\s*)- iframe \[ref=(.*)\]/);
+    const match = line.match(/^(\s*)- iframe (?:\[active\] )?\[ref=(.*)\]/);
```

Comment:
revert?

Reply:
Without it, this code won't go into the content of the iframe.